configure/embeddings/embeddings.Deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     spec:
       containers:
         - name: embeddings
-          image: index.docker.io/sourcegraph/embeddings:insiders@sha256:b68cade0fdf2afcd8e78fbcb44d25bcecaec448f0c6e29fa716834cb95aa8a47
+          image: index.docker.io/sourcegraph/embeddings:insiders@sha256:6bfffb3f0d15a4f71d70989656a35d617021574b9081304602e310436ce9f3f9
```

Comment:
for follow-up: is it write that we don't deploy a tagged version here? cc @stefanhengl

Reply:
This should definitely have a tag. Maybe we need to update the list of images?